packages/web-app-files/src/App.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <main id="files" class="oc-flex oc-height-1-1">
+  <main id="files" class="oc-flex oc-height-1-1" @dragenter="onDragEnter" @mouseleave="onDragLeave">
```

Comment:
Why `@mouseleave` instead of `@dragend`? With `@mouseleave` we only hide the drag area when the mouse pointer actually leaves the `main` tag (e.g. moves over the left sidebar).

Reply:
Ah, because the `dragend` event doesn't fire... weird. Could you look into that? `@mouseleave` is not sufficient, since the drag area stays visible until you move the mouse out of the `main` element.

Reply:
ah okay now i get it, dragend is probably blocked by the browser when handling files. Stackoverflow recommends a clear timeout.

Reply:
Fine with that if UX feels good

Reply:
ok dragover and every other drag event wont work for some reason.